common/src/chain/tokens/mod.rs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://spdx.org/licenses/MIT
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Author(s): A. Sinitsyn
+
+use serialization::{Decode, Encode};
+
+use crate::primitives::Amount;
+
+#[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord, Encode, Decode)]
+pub enum OutputValue {
+    Coin(Amount),
+    Token(TokenData),
```

Comment:
I think it's better not to have this here. We want to see all the modifications related to this when it's added later.

Reply:
Agreed. Let's have just one variant here for now, and 1-way matches where `OutputValue` is being analysed. When you introduce a new variant later on, the places that need updating will be flagged up by the compiler.